frontend/PULL_REQUEST_TEMPLATE

```diff
@@ -2,17 +2,19 @@ Description of PR that completes issue here...
 
 ## Changes
 
-- Item 1
-- Item 2  
+<name_of_repository>:
```

Comment:
Curious, why include repo name here?

Reply:
That's a great question @Gabriel-Martin! I think this was introduced when we began using monorepos, so we could quickly scan and see which portions of the application were changed.

Outside of a monorepo, it could be useful when changes to an outside repository are relevant. Otherwise, you're right, it's redundant 😆 I'll go ahead and remove it for the purposes of this template.

Reply:
Ah, that makes sense! I hadn't thought of the monorepo situation (never worked in one). Thanks!